src/SFA.DAS.AssessorService.Application.Api.Client/Azure/AzureApiClientBase.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using SFA.DAS.AssessorService.Application.Api.Client.Clients;
+using SFA.DAS.AssessorService.Application.Api.Client.Exceptions;
+
+namespace SFA.DAS.AssessorService.Application.Api.Client.Azure
+{
+    public abstract class AzureApiClientBase : IDisposable
```

Comment:
We seem to have the same class used in a few projects now - maybe we should rationalise this?